doc/manual/src/protocols/json/store-object-info.md

```diff
@@ -37,28 +39,30 @@ Info about a [store object].
 These are not intrinsic properties of the store object.
 In other words, the same store object residing in different store could have different values for these properties.
 
-* `deriver` (optional):
+* `deriver`:
 
-  The path to the [derivation] from which this store object is produced.
+  If known, the path to the [derivation] from which this store object is produced.
```

Comment:
```suggestion
  If known, the path to the [derivation] from which this store object was produced.
```

(If you don't mind,) present tense would suggest this is a property rather than an event. It's an event, which happened in the past.